common/reducers/config/networks/staticNetworks.ts

```diff
@@ -23,12 +23,16 @@ export type State = { [key in StaticNetworkIds]: StaticNetworkConfig };
 // Must be a website that follows the ethplorer convention of /tx/[hash] and
 // address/[address] to generate the correct functions.
 // TODO: put this in utils / libs
-export function makeExplorer(name: string, origin: string): BlockExplorerConfig {
+export function makeExplorer(
+  name: string,
+  origin: string,
+  addressPath: string = 'address'
```

Comment:
I'd like to see this become a config, rather than a bunch of misc. args, but I'd say that can come the next time we need to make an explorer more configurable.